subsys/mgmt/suitfu/Kconfig

```diff
@@ -4,17 +4,22 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
-menu "SUIT-based Firmware Update Management"
-
-config MGMT_SUITFU
-	bool "Serial SUIT Firmware Update support"
+menuconfig MGMT_SUITFU
+	bool "SUIT-based Firmware Update Management"
 	depends on MCUMGR
 	depends on SSF_SUIT_SERVICE_ENABLED
 	depends on FLASH
 	depends on MCUMGR_SMP_LEGACY_RC_BEHAVIOUR
 	select MCUMGR_SMP_CBOR_MIN_DECODING_LEVEL_5
 	select MCUMGR_SMP_CBOR_MIN_ENCODING_LEVEL_3
 
+if MGMT_SUITFU
+
+config MGMT_SUITFU_INITIALIZE_SUIT
+	bool "Initialize the SUIT DFU library"
```

Comment:
"Initialize the SUIT DFU library on startup"?

Could you add a "help" section to explain more what this actually does and that if this is not set the user must initialize the library manually?